README.md

```diff
@@ -44,4 +44,7 @@ See [this](https://github.com/angular/angular/blob/master/README.md) for up-to-d
 
 ## Contributing to the project
 
+**NOTE: We appreciate all the interest and enthusiasm, but this project is just in such a volatile state right now that we are not going to accept pull requests that adds new component or features.
```

Comment:
"...not going to accept pull requests that add new components...".  sorry, i accidentally committed the previous comment before the grammar registered.